easybuild/easyblocks/p/pytorch.py

```diff
@@ -49,9 +50,10 @@ class EB_PyTorch(PythonPackage):
     def extra_options():
         extra_vars = PythonPackage.extra_options()
         extra_vars.update({
-            'excluded_tests': [{}, 'Mapping of architecture strings to list of tests to be excluded', CUSTOM],
-            'custom_opts': [[], 'List of options for the build/install command. Can be used to change the defaults ' +
-                                'set by the PyTorch EasyBlock, for example ["USE_MKLDNN=0"].', CUSTOM],
+            'custom_opts': [[], "List of options for the build/install command. Can be used to change the defaults " +
+                                "set by the PyTorch EasyBlock, for example ['USE_MKLDNN=0'].", CUSTOM],
+            'excluded_tests': [{}, "Mapping of architecture strings to list of tests to be excluded", CUSTOM],
+            'max_failed_tests': [10, "Maximum number of failing tests", CUSTOM],
```

Comment:
I would set the default to 0 as e.g. existing PyTorch ECs don't have any failed, non-excluded tests (or should at least).
Also having to set this explicitly in the EC makes it clear that there is something "fishy" with that specific PyTorch version.

Reply:
@casparvl Thoughts on this?

Reply:
I agree with @Flamefire here, we should _know_ what the failing tests are and how many, putting the number in the easyconfig file is a way of ensuring that it at least gets documented.

Reply:
fixed in 82011bf